test/EcdsaMultikey.spec.js

```diff
@@ -171,7 +172,11 @@ describe('EcdsaMultikey', () => {
 
 function _ensurePublicKeyEncoding({keyPair, publicKeyMultibase}) {
   keyPair.publicKeyMultibase.startsWith(MULTIBASE_BASE58_HEADER).should.be.true;
-  const decodedPubkey = base58.decode(publicKeyMultibase);
-  const encodedPubkey = base58.encode(decodedPubkey);
+  const decodedPubkey = base58.decode(publicKeyMultibase.slice(1));
+  const ecdsaCurve = getNamedCurveFromPublicMultikey({
```

Comment:
```suggestion
  if(publicKeyMultibase[0] !== 'z') {
    throw new Error('"publicKeyMultibase" must be multibase-base58-encoded.');
  }
  const decodedPubkey = base58.decode(publicKeyMultibase.slice(1));
  const ecdsaCurve = getNamedCurveFromPublicMultikey({
```

Reply:
Well, that line is checking `keyPair.publicKeyMultibase`. I would also be fine with duplicating that line and applying it to `publicKeyMultibase`.

Reply:
Ah OK, just made that change for consistency 👍